src/master_element/MasterElement.C

```diff
@@ -641,6 +641,317 @@ HexSCS::ipNodeMap(
   return &ipNodeMap_[ordinal*4];
 }
 
+
+//--------------------------------------------------------------------------
+//-------- shape_fcn -------------------------------------------------------
+//--------------------------------------------------------------------------
+void
+HexSCS::shape_fcn(SharedMemView<DoubleType**> &shpfc)
```

Comment:
@spdomin Any specific reason to maintain this function call instead of implementing the body of `hex8_shape_fcn` here?